Dockerfile

```diff
@@ -0,0 +1,55 @@
+# https://docs.docker.com/samples/rails/
+FROM ruby:2.7.3
```

Comment:
Latest v2 ruby patch version is 2.7.4

Reply:
This version was chosen due to the gem file, is there some way to update the gem file?

Reply:
Update the version in the `Gemfile` and `.ruby-version`, then run `bundle` and commit the changes.  If `bundle exec rake` passes then you're good to go.